mods/ra/chrome/editor.yaml

```diff
@@ -394,3 +394,9 @@ Container@EDITOR_WORLD_ROOT:
 			Align: Left
 			Font: Bold
 			Contrast: true
+		Button@EYEDROPPER_BUTTON:
+			X: 665
+			Width: 120
+			Height: 25
+			Text: Eyedropper tool
```

Comment:
This label is too long, and overlaps the minimap.  I suggest renaming this to "Select terrain" and moving the button.